### PR TITLE
Add 46elks doc

### LINKED ISSUE #115: Add FortySixElks sms provider documentation

### PR DESCRIPTION
**Pull request by ShatilKhan**
Fixes #115 
Added 46elks SMS provider doc.

### CODE REVIEW

File: channels-and-providers/sms/46elks.mdx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: '46elks'
+description: 'Learn how to use the 46elks provider to send sms notifications using Novu'
+---
+
+[46elks](https://46elks.com/) API let's you intigrate SMS, MMS and phone calls into your applications with just a few lines of code.
```
Comment (unicodeveloper) [inline channels-and-providers/sms/46elks.mdx:6]: Please rephrase to:


You can use the [46elks](https://46elks.com/) provider to send SMS messages to your customers using the Novu platform with a single API to create multi-channel experiences.

File: channels-and-providers/sms/46elks.mdx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+# Getting Started
+
+To use the 46elks provider in the sms channel, you will need to create a 46elks account and add your `API Username` & `API Password` to the 46elks integration on the Novu platform.
+
+# Creating the 46elks integration with Novu
+
+- Visit the [Integrations](https://web.novu.co/integrations) page on Novu.
+- Select 46elks service
```
Comment (unicodeveloper) [inline channels-and-providers/sms/46elks.mdx:15]: The second step is to click on "Add a provider" before selecting 46elks provider. So please add that.

**Review by unicodeveloper — COMMENTED** [review]
Thanks for your contribution. Please attend to the comments.

File: channels-and-providers/sms/46elks.mdx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+- Click `Next`
+- Choose your preferred deployment environment `Development`\`Production`. Then click `CREATE`.
```
Comment (unicodeveloper) [inline channels-and-providers/sms/46elks.mdx:17]: => Choose your preferred deployment environment: `Development` or `Production`. Then Click the `Create` button.

**Comment by unicodeveloper** [comment]
@ShatilKhan Please any updates here?

**Comment by ShatilKhan** [comment]
Done with the update @unicodeveloper 
I'm working on my other PRs to finish up

**Review by unicodeveloper — APPROVED** [review]

**Comment by unicodeveloper** [comment]
We greatly appreciate your contribution. We hope you'll stay with us and continue to be an integral part of this project.

Please consider giving this repo a star and joining our community discord server, if you haven't already, to access more resources and connect with like-minded individuals. [Discord Community here 🎉 ](https://discord.gg/novu) #Hacktoberfest

**Pull request merged**